src/worker/mod.rs

```diff
@@ -277,16 +282,32 @@ impl Worker {
             }?;
 
             match selected_f {
-                Some(work) => match self.apply_server_work(work).await? {
-                    NewWfTaskOutcome::IssueActivation(a) => return Ok(a),
-                    NewWfTaskOutcome::TaskBuffered => {
-                        // If the task was buffered, it's not actually outstanding, so we can
-                        // immediately return a permit.
-                        self.return_workflow_task_permit();
+                Some(work) => {
+                    WORKFLOW_TASK_QUEUE_POLL_SUCCEED_COUNTER.add(
+                        1,
+                        &[KeyValue::new(
```

Comment:
Is the task queue relevant to this metric?

Reply:
Yeah, but these are here just as an example. I'm going to flesh them out next.